<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersDialog</class>
 <widget class="QDialog" name="UsersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabWidget{
    background-color:#000000;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Load Projects</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>801</width>
     <height>411</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Users</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>751</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabView{
                    background-color:#1d1c11;
                }
                </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Groups</string>
    </attribute>
    <widget class="QTableView" name="tableView_Groups">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>751</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoadclients">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Groups Clients</string>
     </property>
    </widget>
    <widget class="QFrame" name="frameTheme">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>751</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Clients</string>
    </attribute>
    <widget class="QTableView" name="tableView_Clients">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>751</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Projects</string>
    </attribute>
    <widget class="QTableView" name="tableView_Projects">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>751</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonLoadusers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Users</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGroups">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxClients">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxProjects">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>530</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>600</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
